src/api/app/models/branch_package.rb

```diff
@@ -185,7 +186,7 @@ def create_branch_packages(tprj)
            p[:package].project != p[:link_target_project]
           opts[:extendvrev] = '1'
         end
-        tpkg.branch_from(oproject, opackage, opts)
+        tpkg.branch_from(oproject, opackage, opts) unless scmsync_active
```

Comment:
One thing I wonder about, what happens if I branch the scmsync package outside the SCM/CI feature now? Nothing?

Reply:
@adrianschroeter Branching a scmsync package is supported or not?

Reply:
@hennevogel It would branch the package, but doesnt copy the files. So since its not really clear if it should be this way generally, maybe I only do it for the workflows for now?

Reply:
No this makes sense in general if `tpkg` gets created.